pkgs/applications/misc/onboard/default.nix

```diff
@@ -0,0 +1,157 @@
+{ config
```

Comment:
What is this for?

Reply:
I I think that remains from when I was using it as one of my own custom packages. I'll remove it.